JDI.Light/JDI.Light/Elements/Init/EntitiesCollection.cs

```diff
@@ -0,0 +1,42 @@
+﻿using JDI.Light.Interfaces.Base;
+using JDI.Light.Interfaces.Composite;
+using System;
+using System.Collections.Generic;
+
+namespace JDI.Light.Elements.Init
+{
+    public class EntitiesCollection
+    {
+        protected EntitiesCollection()
+        {
+        }
+
+        public static Dictionary<string, IPage> Pages { get; set; } = new Dictionary<string, IPage>();
+        public static Dictionary<string, List<IBaseElement>> Elements { get; set; } = new Dictionary<string, List<IBaseElement>>();
+
+        public static T getPage<T>(string pageName, T type)
+        {
+            var page = getPage(pageName);
+            if (page != null && page.GetType().IsClass)
+                return (T)page;
+            throw new Exception($"Can't cast element {page.GetType()} to {type}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['System.Exception' should not be thrown by user code.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5509587)